modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1052,38 +1053,41 @@ function DOMUtils(doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
 
   const createRng = () => doc.createRange();
 
-  const split = (parentElm: Node, splitElm: Node, replacementElm?: Node) => {
-    let r = createRng(), bef, aft, pa;
+  const split = <T extends Node>(parentElm: Node, splitElement: T, replacementElement?: T): T => {
+    let range = createRng();
+    let beforeFragment: DocumentFragment;
+    let afterFragment: DocumentFragment;
+    let parentNode: Node;
 
-    if (parentElm && splitElm) {
+    if (parentElm && splitElement) {
```

Comment:
Why'd you rename this, but not `parentElm`? It's also pretty common for us to shorten `Element` to either `elm` or `elem`.

Reply:
Because I changed the names of variables in one area ( down at the `r` and `bef` ), and didn't really check that I changed all variables. Scanned quickly, but `parentElm` was long enough that I didn't immediately notice it as a hard-to-read short.
Can go either way on the Elms, I'd favor Element, but if it's more in line with the rest of the codebase I can change it back to elms.

Reply:
I'd favour a shorter, but still readable version, as it causes less issues with code wrapping (particularly on small screens, such as laptops).